lib/kennel/github_reporter.rb

```diff
@@ -24,7 +24,7 @@ def initialize(token, ref: "HEAD")
     end
 
     def report(&block)
-      output = Utils.strip_shell_control(Utils.tee_output(&block).strip)
```

Comment:
this is removed because it won't have shell-control anymore ?

Reply:
Yup, `tee_output` provides a non-tty, so won't have any escape sequences to remove.